### PR TITLE
EVAKA-HOTFIX: update codegen

### PR DESCRIPTION
**Pull request by Joosakur**

### CODE REVIEW

**Review by mikkopiu — APPROVED** [review]

File: frontend/src/lib-common/generated/api-types.d.ts [hunk 1]
```diff
@@ -3263,6 +3286,11 @@ export interface Coordinate {
...
...Child.CREATE_ASSISTANCE_NEED' | 'fi.espoo.evaka.shared.security.Action$Child.READ_ASSISTANCE_NEED' | 'fi.espoo.evaka.shared.security.Action$Child.CREATE_BACKUP_CARE' | 'fi.espoo.evaka.shared.security.Action$Child.READ_BACKUP_CARE' | 'fi.espoo.evaka.shared.security.Action$Child.CREATE_BACKUP_PICKUP' | 'fi.espoo.evaka.shared.security.Action$Child.READ_BACKUP_PICKUP' | 'fi.espoo.evaka.shared.security.Action$Child.READ_DAILY_SERVICE_TIMES' | 'fi.espoo.evaka.shared.security.Action$Child.UPDATE_DAILY_SERVICE_TIMES' | 'fi.espoo.evaka.shared.security.Action$Child.DELETE_DAILY_SERVICE_TIMES' | 'fi.espoo.evaka.shared.security.Action$Child.READ_PLACEMENT' | 'fi.espoo.evaka.shared.security.Action$Child.READ_FAMILY_CONTACTS' | 'fi.espoo.evaka.shared.security.Action$Child.UPDATE_FAMILY_CONTACT' | 'fi.espoo.evaka.shared.security.Action$Child.READ_GUARDIANS' | 'fi.espoo.evaka.shared.security.Action$Child.READ_FEE_ALTERATIONS' | 'fi.espoo.evaka.shared.security.Action$Child.READ_CHILD_RECIPIENTS' | 'fi.espoo.evaka.shared.security.Action$Child.UPDATE_CHILD_RECIPIENT' | 'fi.espoo.evaka.shared.security.Action$Child.CREATE_VASU_DOCUMENT' | 'fi.espoo.evaka.shared.security.Action$Child.READ_VASU_DOCUMENT'
```
Comment (jaahon) [inline frontend/src/lib-common/generated/api-types.d.ts:3292]: 🤔

Comment (Joosakur) [reply frontend/src/lib-common/generated/api-types.d.ts:3292]: huh.. thats a bug

Comment (Joosakur) [reply frontend/src/lib-common/generated/api-types.d.ts]: fixed

Comment (Gekkio) [reply frontend/src/lib-common/generated/api-types.d.ts]: Or is it?

```
     permittedActions: Child[]
```

```
/**
* Generated from fi.espoo.evaka.attendance.Child
*/
export interface Child {
```

:thinking:

Comment (Joosakur) [reply frontend/src/lib-common/generated/api-types.d.ts]: asdf

Comment (Joosakur) [reply frontend/src/lib-common/generated/api-types.d.ts]: Hmm this needs more fixing, need to somehow make sure that we don't confuse classes with same name.. I'll just exclude the classes using permittedActions for now to get master build working

Comment (Joosakur) [reply frontend/src/lib-common/generated/api-types.d.ts]: Not sure if we should revert the whole thing for now

**Pull request merged**